test/testdata/nettemplates/perf/mkconf.py

```diff
@@ -33,10 +34,20 @@
 
   d["Nodes"].append(node)
 
+npn_nodes = 50
+for n in range(0, npn_nodes):
+  node = {
+    "Name": "NPNode%d" % n,
+    "Wallets": [],
+    "DeadlockDetection": -1,
+    "ConfigJSONOverride": '{"ForceFetchTransactions":true}'
```

Comment:
This is what the change is really about. So, why this?

Reply:
I used mkconf to generate local topologies for some tests, and decided to leave ConfigJSONOverride example there. I can remove if it does not make much sense having it here.